src/is-my-room/index.ts

```diff
@@ -0,0 +1,25 @@
+import {isRoomVisible} from '@open-screeps/is-room-visible'
+
+export function isMyRoom(roomName: string | Room): boolean{
+  if(!roomName){
```

Comment:
This isn't really necessary, since TypeScript doesn't allow falsy values here.
The published declaration works as documentation in some way already (as long as we don't have proper documentation setup), if the user is unable to provide correct data that shouldn't be our concern.